RealEstate/apps/core/views.py

```diff
@@ -45,3 +46,83 @@ def get(self, request, *args, **kwargs):
         couple = Couple.objects.filter(homebuyer__user=request.user)
         house = House.objects.filter(couple=couple)
         return render(request, 'core/homebuyerHome.html', {'couple': couple, 'house': house})
+
+
+class EvalView(BaseView):
+    """
+    View for the Home Evaluation Page. Currently, this page is decoupled
+    from the rest of the app and uses static elements in the database.
+    """
+    def get(self, request, *args, **kwargs):
+        homebuyer = Homebuyer.objects.filter(user_id=request.user.id)
```

Comment:
I added a method to the User object which gives an easier way to the homebuyer.
homebuyer = request.user.role_object

This property will return the Homebuyer instance if the user is a homebuyer, or the Realtor instance if they are a realtor (or None if neither).